woocommerce-services.php

```diff
@@ -809,7 +809,6 @@ public function load_dependencies() {
 			$shipping_label        = new WC_Connect_Shipping_Label( $api_client, $settings_store, $schemas_store, $payment_methods_store );
 			$nux                   = new WC_Connect_Nux( $tracks, $shipping_label );
 			$taxjar                = new WC_Connect_TaxJar_Integration( $api_client, $taxes_logger, $this->wc_connect_base_url );
-			$options               = new WC_Connect_Options();
```

Comment:
During this PR I noticed that this variable was unused, so just sneaking in the update.